### PR TITLE
Freeipmi yaml updates

### PR DESCRIPTION
**Pull request by Ancairon**
##### Summary

- Consolidate to one module
- Add description, category, keywords, icon

### CODE REVIEW

File: collectors/freeipmi.plugin/multi_metadata.yaml [hunk 1]
```diff
@@ -1,94 +1,26 @@
...
-                - name: events
   - meta:
       plugin_name: freeipmi.plugin
       module_name: sensors
       monitored_instance:
-        name: freeipmi sensors
-        link: ''
-        categories: []
-        icon_filename: ''
+        name: FreeIPMI
```
Comment (shyamvalsan) [inline collectors/freeipmi.plugin/multi_metadata.yaml:7]: Should this be FreeIPMI? Or just IPMI? 

I mean users care about IPMI don't they?

Comment (Ancairon) [reply collectors/freeipmi.plugin/multi_metadata.yaml:7]: That's a good question, plugin is called `freeipmi`, so I think this is one for @ilyam8 mostly..

Comment (shyamvalsan) [reply collectors/freeipmi.plugin/multi_metadata.yaml:7]: cc: @ktsaou @sashwathn

Comment (Ferroin) [reply collectors/freeipmi.plugin/multi_metadata.yaml:7]: Name should be IPMI, but we should have a ‘virtual’ integration called ‘FreeIPMI’ as well.

**Review by sashwathn — APPROVED** [review]

File: collectors/freeipmi.plugin/multi_metadata.yaml (hunk 1 above)
Comment (Ancairon) [reply collectors/freeipmi.plugin/multi_metadata.yaml:7]: right, will spin up an anchor for this, commit incoming.

**Comment by Ancairon** [comment]
didn't have an icon for freeipmi, so used a generic netdata one.

**Review by sashwathn — APPROVED** [review]

**Review by sashwathn — APPROVED** [review]

**Pull request merged**